Native-Frameworks/FrameworkGridView.swift

```diff
@@ -23,18 +23,22 @@ struct FrameworkGridView: View {
                     .navigationTitle("🍎 Frameworks")
                     .navigationBarItems(trailing: EllipsisButton(isPressed: $isPressed))
                     .fullScreenCover(isPresented: $viewModel.isShowingDetailView) {
-                        FrameworkDetailView(isShowingDetailView: $viewModel.isShowingDetailView, framework: viewModel.selectedFramework ?? MockData.sampleFramework)
+                        FrameworkDetailView(
+                            isShowingDetailView: $viewModel.isShowingDetailView,
+                            framework: viewModel.selectedFramework
+                        )
                     }
                 }
                 
             }
             //TODO: Refactor logic to viewModel
+            // Note: Its view and it don't and I dont see any extra logic, but probably you can move it to separete property
             if isPressed {
                 GeometryReader { geo in
                     PopupSelectionView(isGridviewSelected: $isGridviewSelected, isListviewSelected: $isListviewSelected)
                 }
                 .offset(x: UIScreen.screenWidth * 0.61, y: UIScreen.screenHeight * 0.1)
```

Comment:
```
GeometryReader { reader in
 PopupSelectionView(isGridviewSelected: $isGridviewSelected, isListviewSelected: $isListviewSelected)		                  
                  .offset(x: reader.size.width * 0.61, y: reader.size.height * 0.1)
}
```
In the current case your reader sizes will be size of your popup, so the width should work, I hope,  If you want the size of FrameworkGridView you should wrap all your content in GeometryReader

Reply:
If you check latest commit and using this solution - it works. Let me know when it looks good to merge

Reply:
If it works on all devices I accept it :) please try it on SE 12 and 12 Pro